### ISSUE TITLE
Third party integrations not registered in admin

### ISSUE THREAD

**Issue by hasier** [MaistrenkoAnton/django-material-admin#49]
Great work on materializing Django admin, it's brilliant! I am quite intensely using it in one of my projects and the result is great.

I have found the following issue with third party packages, and I am not sure I am doing something wrong or if it is an actual issue. If you can confirm this behaviour is incorrect, I am happy to try to fix it and submit a PR with it.

### Issue
Third party packages using admin are not displayed in the index of admin, and their admin views cannot be accessed. This can be reproduced with a very simple setup:
- `pip install django django-material-admin django-invitations`
- Add a basic settings file and add the following apps:
```python
INSTALLED_APPS = [
    'material.admin',
    'material.contrib.admin',
    'material.contrib.sites',
    ...
    'invitations',
    ...
]
```
- Add the following to `urls.py`:
```python
urlpatterns = [
    path('admin/', include('material.admin.urls')),
    path('invitations/', include('invitations.urls', namespace='invitations')),
    ...
]
```
- Run the server.
- The admin index won't display the `Invitation` model, which is registered by the `django-invitations` package.
- This same setup works removing `material.admin` and adding `django.contrib.admin` instead.

### Possible cause
I believe this is happening because `django-material-admin` uses its own implementation of `AdminSite`. Even if it inherits from `AdminSite`, the package only uses `material.admin.sites.site` to register admin models. And the recommendation for custom admin views is to use `material.admin.decorators.register`, which also uses `material.admin.sites.site`.

The problem is that third party packages will always use `django.contrib.admin.site` to register their admin models. As `django-material-admin` is not using this main site, it does not display all their admin views.

### (Potential) solution
I believe that simply removing `material.admin.sites.site` and adding the following in the `settings` file would work.
```python
from django.contrib.auth.apps import AuthConfig
AdminConfig.default_site = 'material.admin.sites.MaterialAdminSite'
```
This should also work with third party packages because Django would be hooked to the custom `MaterialAdminSite` and all the apps that register to admin will use this one.

**Comment by nishat0403** [comment]
> ### (Potential) solution
> I believe that simply removing `material.admin.sites.site` and adding the following in the `settings` file would work.
> 
> ```python
> from django.contrib.auth.apps import AuthConfig
> AdminConfig.default_site = 'material.admin.sites.MaterialAdminSite'
> ```
> 
> This should also work with third party packages because Django would be hooked to the custom `MaterialAdminSite` and all the apps that register to admin will use this one.

Can you please elaborate a little more on this solution?
From where do I remove `material.admin.sites.site`
and where do I add the following

```
from django.contrib.auth.apps import AuthConfig
AdminConfig.default_site = 'material.admin.sites.MaterialAdminSite'
```

**Comment by MaistrenkoAnton** [comment]
Thanks for the report!
You are right, the 3rd party packages register admins to regular admin site. I'm currently working on it  in 1.6.X version of package, soon it will be available.

**Comment by MaistrenkoAnton** [comment]
Fixed since published version `v1.6.0`. Please recheck the documentation after installing. Configurations were slightly changed.

**Comment by hasier** [comment]
Works perfectly, thanks for the lightning response! Keep on rocking! 🤘
